pages/cloudflare/caching.mdx

```diff
@@ -212,6 +212,22 @@ For staging, when your site receives low traffic from a single IP, you can repla
 
 ### References
 
+#### Static Assets Caching
+
+<Callout type="info">
+  The worker doesn't run **in front** of static assets, so `next.config.js` `headers` option doesn't apply to
+  public files (`public`) and static build files (like `_next/static`).
+</Callout>
+
+By default, Cloudflare [Static Assets headers](https://developers.cloudflare.com/workers/static-assets/headers/#default-headers) don't enable browser caching. To add browser cache headers or any custom headers to Static Assets, create a `public/_headers` file:
```

Comment:
We should add that Next default is [the same](https://nextjs.org/docs/pages/api-reference/file-conventions/public-folder#caching) because the files can change across deployment.

It is important to warn users that file that might change across deployments should not have a big `max-age`